### ISSUE TITLE
[Ops] [Optim] Update to 1.9.

### ISSUE THREAD

**Issue by wangshangsam** [UofT-EcoSystem/hfta#37]
Ops:
- [x] batchnorm.py
- [x] conv.py
- [x] [no change needed] dropout2d.py
- [x] embedding.py
- [x] layernorm.py
- [x] linear.py
- [ ] [no change needed?] multiheadattention.py
- [ ] pool.py
- [ ] transformerencoderlayer.py

Optim:
- [ ] adadelta.py
- [ ] adam.py
- [ ] lr_scheduler.py
